app/models/location.py

```diff
@@ -29,20 +38,18 @@ def create_zip_code(zip_code):
         """
         result = ZIPCode.get_by_zip_code(zip_code)
         if result is None:
-            result = ZIPCode(zip_code)
+            result = ZIPCode(zip_code=zip_code)
```

Comment:
we can leave this line as `result = ZIPCode(zip_code)` because we didn't end up changing the input to `__init__`